docs/en/query_language/create.md

```diff
@@ -299,4 +299,145 @@ Depending on dictionary [layout](dicts/external_dicts_dict_layout.md) one or mor
 
 For more information, see [External Dictionaries](dicts/external_dicts.md) section.
 
+## CREATE USER {#create-user-statement}
+
+Creates a [user account](../operations/access_rights.md#user-account-management).
+
+!!! info "Note"
+    Some of the access control functionality is available through the [users.xml](settings/settings_users.md) server configuration files, but we don't recommend using this way of user permissions management.
+
+### Syntax {#create-user-syntax}
+
+```sql
+CREATE USER [IF NOT EXISTS | OR REPLACE] name
+    [IDENTIFIED [WITH {NO_PASSWORD|PLAINTEXT_PASSWORD|SHA256_PASSWORD|SHA256_HASH|DOUBLE_SHA1_PASSWORD|DOUBLE_SHA1_HASH}] BY {'password'|'hash'}]
+    [HOST {LOCAL | NAME 'name' | NAME REGEXP 'name_regexp' | IP 'address' | LIKE 'pattern'} [,...] | ANY | NONE]
+    [DEFAULT ROLE role [,...]]
+    [SETTINGS variable [= value] [MIN [=] min_value] [MAX [=] max_value] [READONLY|WRITABLE] | PROFILE 'profile_name'] [,...]
+```
+
+#### Identification
+
+There are multiple ways of user identification:
+
+- `IDENTIFIED WITH no_password`
+- `IDENTIFIED WITH plaintext_password BY 'qwerty'`
+- `IDENTIFIED WITH sha256_password BY 'qwerty'` or `IDENTIFIED BY 'password`
+- `IDENTIFIED WITH sha256_hash BY 'hash'`
+- `IDENTIFIED WITH double_sha1_password BY 'qwerty'`
+- `IDENTIFIED WITH double_sha1_hash BY 'hash'`
+
+#### User Host
+
+User host is a host from which a connection to ClickHouse server could be established. Host can be specified in the `HOST` section of query by the following ways:
+
+- `HOST IP 'IP_ADDRESS_OR_SUBNETWORK'` — User can connect to ClickHouse server only from the specified IP address or a [subnetwork](https://en.wikipedia.org/wiki/Subnetwork). For example, `HOST IP '192.168.0.0/16'`.
```

Comment:
The capital letters in `IP_ADDRESS_OR_SUBNETWORK` look like this is a keyword.
And it seems it would be useful to write more examples here, e.g. for IPv6 addresses/subnets.